tests/Examples/swift-bindings/core/basic.txt

```diff
@@ -0,0 +1,7 @@
+# Check basic 'core' functionality of the swift bindings
+#
+# RUN: xcrun -sdk macosx swiftc %{srcroot}/examples/swift-bindings/core/basic.swift -I %{srcroot}/build/products/swift-bindings -L %{llbuild-lib-dir} -Xcc -fmodule-map-file=%{srcroot}/products/libllbuild/public-api/llbuild/module.modulemap -emit-executable %{srcroot}/build/products/swift-bindings/CMakeFiles/llbuild.swift.o -lllbuild -o %t.exe
```

Comment:
This should define and use a new `lit` substitution variable for the path to `swiftc`. We also obviously will need to update it for how the bindings get built.